torchdynamo/symbolic_convert.py

```diff
@@ -898,6 +898,79 @@ def BUILD_STRING(self, inst):
             result = str_var.value + result
         self.push(ConstantVariable(value=result))
 
+    def IS_OP(self, inst):
+        left, right = self.popn(2)
+        options = VariableTracker.propagate([left, right])
+        op = inst.argval
+        if (
```

Comment:
Is this code copy and pasted from `COMPARE_OP`?  Could we just call `COMPARE_OP` to reuse code?